assets/sass/local.sass

```diff
@@ -344,3 +344,31 @@ dt.summary-list__key__wider
 
 .text-align-center
   text-align: center !important
+
+.margin-top-bottom-0
+  margin-top: 0
+  margin-bottom: 0
+
+.print-link 
+    display: inline-block
+    margin: 0 0 15px -10px
```

Comment:
Think. it might generally be good practice to use [gov uk spacings](https://design-system.service.gov.uk/styles/spacing/) rather than hard coding pixel sizes:

Reply:
Good idea. 
done.